gcp/modules/gcp-stackdriver-alerting/resources/notification_channels/alerts_at_raisingthefloor_org.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "email",
+  "labels": {
+    "email_address": "alerts+${domain_name}@raisingthefloor.org"
```

Comment:
Can you document the process of authorizing new senders somewhere, similar to https://github.com/gpii-ops/gpii-infra/blob/master/aws/README.md#alerts-and-notifications?

Reply:
Done.